src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1660,7 +1660,7 @@ static void oauth_cb (rd_kafka_t *rk, const char *oauthbearer_config,
                                                   sizeof(err_msg));
         Py_DECREF(result);
         if (rd_extensions) {
-                for(int i = 0; i < rd_extensions_size; i++) {
+                for(i = 0; i < rd_extensions_size; i++) {
```

Comment:
Since `i` is only used in this scope/block, declare i on the line above instead of at the head of the function